hushline/templates/premium.html

```diff
@@ -0,0 +1,98 @@
+{% extends "base.html" %}
+
+{% block title %}Premium{% endblock %}
+
+{% block content %}
+  <h2>Choose a Hush Line Plan</h2>
+
+  <div id="plan-wrapper">
+    <div class="plan">
+      {% if user.is_free_tier() %}
+        <p class="plan-status badge">📍 Current Plan</p>
+      {% endif %}
+      {% if user.is_business_tier() %}
+        <p class="plan-status">Fewer Features</p>
+      {% endif %}
+      {% include "premium/free-features.html" %}
+    </div>
+    <div class="plan plan-recommended">
+      {% if user.is_free_tier() %}
+        <p class="plan-status badge">⭐️ Recommended</p>
+      {% endif %}
+      {% if user.is_business_tier() %}
+        <p class="plan-status">Current Plan</p>
+      {% endif %}
+      {% include "premium/business-features.html" %}
+      {% if user.is_free_tier() %}
+        <form
+          id="upgrade-form"
+          action="{{ url_for('premium.upgrade') }}"
+          method="post"
+        >
+          <button id="upgrade">Upgrade Now</button>
+        </form>
+      {% endif %}
+      {% if user.is_business_tier() %}
+        {% if user.stripe_subscription_cancel_at_period_end %}
+        <p>Your subscription will expire on {{ user.stripe_subscription_current_period_end.strftime('%B %d, %Y') }}.</p>
```

Comment:
Nit: This and the next forms aren't indented relative to their enclosing `if/else` block